en_US.json

```diff
@@ -2044,6 +2046,8 @@
 	"reportQuery.page.title.type.reqSummary": "Edge Request Summary",
 	"reportQuery.page.title.type.statusDetails": "Status Code Details Chart",
 	"reportQuery.page.title.type.volSummary": "Edge Traffic Volume Summary Chart",
+	"reportQuery.page.validation.count.group.by": "Please select not more than 2 items.",
+	"reportQuery.page.validation.count.wildcard": "Multiple wildcard hostnames is not allowed.",
```

Comment:
Change the English to:

"Please select no more than two items."
"Multiple wildcard hostnames are not allowed."

Reply:
I changed some text by the suggestions
Thanks.